documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,506 @@
+# Get Started
+
+<!--- ash-hq-hide-start --> <!--- -->
+
+This documentation is best viewed at [ash-hq.org](https://ash-hq.org)
+
+<!--- ash-hq-hide-stop --> <!--- -->
+
+## Goals
+
+In this guide we will:
+
+1. Create a new Phoenix project
+2. setup Ash, AshPhoenix and AshPostgres as dependencies
```

Comment:
Might just wanna capitalise the "S" here for consistency, super nitpicky, sorry!

Reply:
Yea I should, I need to give this whole thing a proof read really. 😅

Reply:
addressed: [5c2cb7f](https://github.com/ash-project/ash/pull/504/commits/5c2cb7fdf6732e6b12681b9cd23819322f0baa87)